web-dev/src/components/UserProfile/UserProfileComponent.js

```diff
@@ -0,0 +1,35 @@
+import "../Profile/profile.css"
+
+const UserProfileComponent = (profile) => {
```

Comment:
Confirm the profile.profile.**attributes** conforms to the attribute names in the user schema.
![image](https://user-images.githubusercontent.com/55962290/165304787-06a12e4c-1277-4539-86a8-090bd61fa01a.png)